e2e-tests/playwright/utils/constants.ts

```diff
@@ -0,0 +1,4 @@
+export const GITHUB_URL = 'https://github.com/';
+export const JANUS_ORG = 'janus-idp';
+export const JANUS_QE_ORG = 'janus-qe';
+export const SHOWCASE_REPO = `${JANUS_ORG}/backstage-showcase`;
```

Comment:
@nilgaar  we have API end points at one place `APIEndpoints.ts` you can consider adding it to `githubAPIEndpoints`

Reply:
I see that majority of endpoints under `githubAPIEndpoints` refer to `backstageShowcaseAPI`. Not sure if my constants fit there. Besides, those constant are intend to be used to avoid misspelling some strings that we may use frequently, and are not indeed endpoints themselves.

Reply:
Intention behind `APIEndpoints.ts` was to have APIEndpoints at one place, I did some refactor in #1588 , Any further  Improvements are welcome.

Reply:
Sounds good! I think we can discuss an standardized approach to code APIs in the project, as we will have Github's, RHDH and we may have more in the future.